"make build" in Docker environment fails

**What happened**:

After running the ./scripts/devenv.sh script on Mac OS X, the Docker instance was successfully created.  Typed in "make build" to create the acs-engine binary, and the build process failed (see output below).

./acs-engine is not available

**What you expected to happen**:

"make build" completes successfully and ./acs-engine is available within Docker container

**How to reproduce it** (as minimally and precisely as possible):

- Clone acs-engine repo on Mac OS X with Docker 17.03.1-ce installed (see below)
- Run ./scripts/devenv.sh
- Docker container is created successfully and interactive session entered
- Run "make build"
- "make build" fails

**Anything else we need to know**:

**Docker Information:**

Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.03.1-ce
 API version:  1.27 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Fri Mar 24 00:00:50 2017
 OS/Arch:      linux/amd64
 Experimental: true

**"make build" output**

root@5d7afc86aad9:/gopath/src/github.com/Azure/acs-engine# make build
go generate -v `go list ./... | grep -v "github.com/Azure/acs-engine/vendor" | sed 's|github.com/Azure/acs-engine|.|g' | grep -v -w '^.$'`
cmd/deploy.go
cmd/generate.go
cmd/root.go
cmd/upgrade.go
cmd/version.go
pkg/acsengine/azureconst.go
pkg/acsengine/const.go
pkg/acsengine/defaults.go
pkg/acsengine/doc.go
pkg/acsengine/engine.go
pkg/acsengine/engine_test.go
pkg/acsengine/fileloader.go
pkg/acsengine/fileloader.go:3: running "go-bindata": exec: "go-bindata": executable file not found in $PATH
pkg/api/apiloader.go
pkg/api/const.go
pkg/api/converterfromapi.go
pkg/api/converterfromupgradeapi.go
pkg/api/convertertoapi.go
pkg/api/convertertoapi_test.go
pkg/api/convertertoupgradeapi.go
pkg/api/doc.go
pkg/api/types.go
pkg/api/types_test.go
pkg/api/upgradeapiloader.go
pkg/api/v20160330/const.go
pkg/api/v20160330/doc.go
pkg/api/v20160330/types.go
pkg/api/v20160330/types_test.go
pkg/api/v20160330/validate.go
pkg/api/v20160930/const.go
pkg/api/v20160930/doc.go
pkg/api/v20160930/types.go
pkg/api/v20160930/types_test.go
pkg/api/v20160930/validate.go
pkg/api/v20170131/const.go
pkg/api/v20170131/doc.go
pkg/api/v20170131/types.go
pkg/api/v20170131/types_test.go
pkg/api/v20170131/validate.go
pkg/api/v20170701/const.go
pkg/api/v20170701/doc.go
pkg/api/v20170701/types.go
pkg/api/v20170701/validate.go
pkg/api/vlabs/const.go
pkg/api/vlabs/doc.go
pkg/api/vlabs/types.go
pkg/api/vlabs/upgradetypesandvalidate.go
pkg/api/vlabs/validate.go
pkg/api/vlabs/validate_test.go
pkg/armhelpers/azureclient.go
pkg/armhelpers/compute.go
pkg/armhelpers/deploymentOperations.go
pkg/armhelpers/deployments.go
pkg/armhelpers/groupsclient.go
pkg/armhelpers/interfaces.go
pkg/armhelpers/mockclients.go
pkg/armhelpers/network.go
pkg/armhelpers/storage.go
pkg/armhelpers/util.go
pkg/armhelpers/util_test.go
pkg/operations/deletevm.go
pkg/operations/scaledownagentpool.go
pkg/operations/scaledownagentpool_test.go
pkg/operations/kubernetesupgrade/upgradecluster.go
pkg/operations/kubernetesupgrade/upgradecluster_test.go
pkg/operations/kubernetesupgrade/upgradeworkflow.go
pkg/operations/kubernetesupgrade/v162upgradeagentnode.go
pkg/operations/kubernetesupgrade/v162upgrademasternode.go
pkg/operations/kubernetesupgrade/v162upgrader.go
test/acs-engine-test/config.go
test/acs-engine-test/config_test.go
test/acs-engine-test/main.go
Makefile:22: recipe for target 'build' failed
make: *** [build] Error 1

--- a/docs/acsengine.md
+++ b/docs/acsengine.md
@@ -16,7 +16,7 @@ same environment used in our Continuous Integration system).
 
 When the execution of `devenv.{ps1,sh}` completes, you should find the console logged into the container. 
 
-Now we need to do a one time call to setup the prerequesits.
+Now we need to do a one-time call to setup the prerequisites.
 
 ```
 make prereqs

--- a/docs/acsengine.md
+++ b/docs/acsengine.md
@@ -14,7 +14,15 @@ This means that you can edit your source code normally in your favorite editor o
 machine, while still being able to compile and test inside of the Docker container (the
 same environment used in our Continuous Integration system).
 
-When the execution of `devenv.{ps1,sh}` completes, you should find the console logged into the container. As a final step, in order to get the `acs-engine` tool ready, you should build the sources with:
+When the execution of `devenv.{ps1,sh}` completes, you should find the console logged into the container. 
+
+Now we need to do a one time call to setup the prerequesits.
+
+```
+make prereqs
+```
+
+As a final step, in order to get the `acs-engine` tool ready, you should build the sources with:
 
 ```
 make build

